jdaviz/configs/default/plugins/viewers.py

```diff
@@ -4,8 +4,18 @@
 
 from jdaviz.core.registries import viewer_registry
 
-__all__ = []
+__all__ = ['JdavizViewerMixin']
```

Comment:
Since the mixin class is not exposed to viewer registery, maybe it does not have to be in `__all__`?

Reply:
tried removing, but this is necessary for API docs to build successfully.

Reply:
OK leave it in then. Thanks for trying!